### DISCUSSION TITLE
Starter Project for MuleSoft based API implementation

### DISCUSSION THREAD

**Discussion by manikmagar** [BattlesnakeOfficial/feedback#225] [Q&A] [answered]
Hi Team!

First of all, thank you for the great gaming environment!

Can I please get some feedback on this mule-battlesnake-starter project - https://github.com/manikmagar/mule-battlesnake-starter?

If that looks as per the expectations, I would love to add it starter projects list - https://docs.battlesnake.com/starter-projects

Some related references: 
- https://github.com/BattlesnakeOfficial/feedback/discussions/200
- https://www.twitch.tv/videos/1510344100

I hope this a right place and category to request this. Thank you for your time!

Regards,
Manik

**Comment by andrewdmaclean** [comment]
Hey @manikmagar 👋 

Taking a look now and will follow-up with next steps!

Cheers,
Andrew

**Comment by andrewdmaclean** [comment, marked as answer]
Hey again @manikmagar 👋 

So as a new MuleSoft and AnyPoint Studio user, I have been struggling through attempting to follow these instructions to get a Battlesnake live. 

Even reviewing Alex's recent streams I am unable to get a Battlesnake running locally, specifically because I can't find a point where she reviews the process of importing a GitHub repository into AnySoft Studio

Perhaps you could provide a more introductory process for importing this project into AnyPoint Studio, including any specialized settings that are required when creating a new project.

**Reply by manikmagar** [reply]
Thank you @andrewdmaclean for reviewing it. You are right. I assumed that the consumer is aware of the MuleSoft ecosystem, which may not be true for any beginner. Alex's stream is for the MuleSoft community and that particular one was to demonstrate Battlesnake, again the assumption was users knew MuleSoft. 

But, your observation totally makes sense. I will add that information and update you back here!

Thank you!

**Reply by manikmagar** [reply]
@andrewdmaclean I have updated the [Quickstart](https://github.com/manikmagar/mule-battlesnake-starter#quickstart) section with references on how to get started on working with Studio and for learning DataWeave language to customize snake behavior. I hope that is more helpful for beginners now. I would appreciate if you could take a look at it again. Looking forward to your feedback on this! Thank you!

**Reply by andrewdmaclean** [reply]
Hey @manikmagar ! 

Thanks so much for the extra context in the docs 🙏 

With those changes I was able to get a Battlesnake up and running pretty quickly!

I will add this to the list of Community Starter Snakes today.

Cheers,
Andrew

**Reply by andrewdmaclean** [reply]
@manikmagar the QuickStart has been added 🥳 

![image](https://user-images.githubusercontent.com/26526271/215158067-32d856e2-635b-4f2c-9789-0d20da442fbb.png)

**Reply by manikmagar** [reply]
This is amazing @andrewdmaclean! Thank you very much and I appreciate your help in getting this added! 🎉
